utils/profiles.ts

```diff
@@ -72,6 +72,7 @@ export const retrievedProfileToExtendedProfile = (
     accepted_tos_version: profile.acceptedTosVersion,
     blocked_inbox_or_channels: profile.blockedInboxOrChannels,
     email: profile.email,
+    is_email_already_taken: false,
```

Comment:
this is always a computed value, so it can't be read from the profile on db
I don't see unwanted effects right now

`UpdateProfile` does't read this info (it computes the `is_email_already_taken` for the updated e-mail)